### PR TITLE
Add a column on protocols.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -289,6 +289,68 @@ <h3>Bluesky Architecture</h3>
...
+                            <li>Python <code>__iter__</code> protocol</li>
+                            <li>Bluesky <code>Msg</code> protocol</li>
+                            <li>Ophyd&ndash;bluesky protocol</li>
+                            <li>Channel Access protocol</li>
+                        </ul>
+                    </section>
+                    <section>
+                        <p class="standout">Numpy <code>__array__</code> protocol</p>
+                        <ul>
+                            <li>Libraries like <em>scikit-image</em> and <em>scikit-learn</em> can share an in-memory array representation withou planning around each other.</li>
```
Comment (mrakitin) [inline index.html]: without

File: index.html [hunk 2]
```diff
@@ -289,6 +289,68 @@ <h3>Bluesky Architecture</h3>
...
+                            <li>An "iterable" is arguably the central concept in Python.</li>
+                            <li>A bluesky experiment procedure ("plan") can be <em>any</em> iterable: a list, a generator (adaptive), or a custom class that implements the <code>__iter__</code> method.</li>
+                            <li>Key point: Rely on what an object can <em>do</em> not what it <em>is</em>.</li>
+                        </ul>
+                    </section>
+                    <section>
+                        <p class="standout">Bluesky <code>Msg</code> protocol</p>
+                        <ul>
+                            <li>An experiment procedure is specified <em>descriptively</em> as a series of messages like:<br /><code>Msg('read', detector)</code></li>
+                            <li>An executor ("Run Engine") consumers the messages and does the actual I/O.</li>
```
Comment (mrakitin) [inline index.html]: consumes

**Review by mrakitin — CHANGES_REQUESTED** [review]
Looks good except a couple of minor typos. Thanks.

**Comment by danielballan** [comment]
Thanks, @mrakitin. Fixed typos and made a couple other small language changes.

**Comment by danielballan** [comment]
Think of this column as backup slides / slides to help us all  get on the same page about what we mean by "protocols" and their value / slides that might be used for very technical audiences. For some talks, we might just include the first two slides and then move on to the next column.

**Review by mrakitin — APPROVED** [review]

**Review by awalter-bnl — APPROVED** [review]
looks good to me

**Pull request merged**